src/Symfony/Component/HttpFoundation/Session/Attribute/NamespacedAttributeBag.php

```diff
@@ -43,7 +43,7 @@ public function __construct($storageKey = '_sf2_attributes', $namespaceCharacter
      */
     public function has($name)
     {
-        $attributes = $this->resolveAttributePath($name);
+        $attributes = &$this->resolveAttributePath($name);
```

Comment:
Then I need to check why analyzer didn't recognize the issue. I'll revert these 2 places, but put some comments regarding mismatches, is it fine?

is_object, is_array are also has no ref safe alternatives, am I right?